lnd.go

```diff
@@ -729,6 +739,9 @@ func Main(lisCfg ListenerCfg) error {
 	// Wait for shutdown signal from either a graceful server stop or from
 	// the interrupt handler.
 	<-signal.ShutdownChannel()
+	if v, ok := chanDbBackend.(kvdb.BackendWithStats); ok {
+		ltndLog.Infof(v.PrintStats())
```

Comment:
Should be removed before this is merged.

Reply:
Removed. Also I reworked the commit stats collector code a little bit to be less coupled with the database.